src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.Generated.cs

```diff
@@ -637,111 +609,111 @@ void IFeatureCollection.Set<TFeature>(TFeature feature)
         {
             if (_currentIHttpRequestFeature != null)
             {
-                yield return new KeyValuePair<Type, object>(IHttpRequestFeatureType, _currentIHttpRequestFeature);
+                yield return new KeyValuePair<Type, object>(typeof(IHttpRequestFeature), _currentIHttpRequestFeature);
```

Comment:
The caching in statics would help a tiny bit here, but it is probably a drop in a bucket given the total cost of the enumerator.

Reply:
Nobody really calls this anyways.